s/strimzi-kafka-bridge/redhat_ubi8/build.sh

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : strimzi-kafka-bridge
+# Version       : master (default)
+# Source repo   : https://github.com/strimzi/strimzi-kafka-bridge
+# Tested on     : RHEL_8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora <krishvoor@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+runtest=0
+
+case "$1" in
+		   # no branch supplied, master is used
+	"")	   BRANCH=""
+		   ;;
+
+	"runtest") BRANCH=""
+		   runtest=1
+		   ;;
+
+	*)	   BRANCH="--branch $1"
+		   if [ "$2" == "runtest" ]
+		   then
+			runtest=1
+     		   fi
+		   ;;
+esac
+
+echo "BRANCH = $BRANCH"
+
+# Clone Repository
+(git clone $BRANCH https://github.com/strimzi/strimzi-kafka-bridge) || (echo "git clone failed" && exit $?)
```

Comment:
Having a parenthesis seems to create a sub-shell and `exit $?` only exits this sub-shell & not overall script which is not what we intend when the git clone fails. Can you please change the exit as below to address this?
`{ echo "git clone $BRANCH failed" && exit $?; }`

For reference, I realized & fixed a similar issue here: - 
https://github.com/ppc64le/build-scripts/pull/896/files#diff-70ea175c74f83cdcbe7c014b79bb4341badc1c18fea54098e0f0b232c71e11b4

Reply:
Thanks for the catch, I have updated the traps accordingly.
Request for review.